llama-index-integrations/tools/llama-index-tools-dappier/llama_index/tools/dappier/real_time_search/base.py

```
@@ -0,0 +1,65 @@
+
+
+class DappierRealTimeSearchToolSpec(BaseToolSpec):
+    """Dappier Real Time Search tool spec."""
+
+    spec_functions = ["search_real_time_data", "search_stock_market_data"]
+
+    def __init__(self, api_key: Optional[str] = None) -> None:
+        """Initialize the Dappier Real Time Search tool spec.
+
+        To obtain an API key, visit: https://platform.dappier.com/profile/api-keys
+        """
```

Ruff (0.8.2)

Multi-line docstring summary should start at the second line

Insert line break and indentation after opening quotes

(D213)